src/components/WwwFrame/PromotionalBanner/Banners/PromoCreditBanner.vue

```diff
@@ -0,0 +1,177 @@
+<template>
+	<div
+		v-if="lendingRewardOffered"
+		class="tw-bg-brand tw-text-white md:tw-py-1.5 md:tw-px-0"
+		v-kv-track-event="['TopNav','click-Promo','Lending Reward Banner']"
+	>
+		Make a Kiva loan <br class="tw-inline">and receive a $25 free credit to lend again.
+	</div>
+	<div v-else class="bonus-banner-holder tw-bg-brand tw-text-center tw-py-1.5 tw-px-5">
```

Comment:
This should only show if the bonus balance is greater than 0, `v-else-if="bonusBalance > 0"`. I think that will mean that you have to add a third element to this component for the `v-else` condition, `<div v-else></div>`, or use v-show instead.

Reply:
Thanks for that.

Original condition is this

               `currentActivePromo() {
			if (this.lendingRewardOffered) {
				return LendingRewardsBanner;
			}
			if (this.bonusBalance > 0) { // TODO: skip if on a checkout/basket page
				return BonusBanner;
			}
			return null;
		},`
		
So, I think is enough with `v-else-if="bonusBalance > 0".`